e2e/gradle/workflow_dispatch/build.gradle.kts

```
@@ -0,0 +1,21 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+import java.io.File
+
+group = "io.github.adamkorcz"
+version = "0.1.47"
```

nit: this is a new package so you could start off with a lower version number
```suggestion
version = "0.0.1"
```